templates/default/flyingduck.hcl.erb

```diff
@@ -2,17 +2,6 @@ services = [
     {
         id = "flyingduck/metrics"
         name = "flyingduck"
-        tags = [
-            "monitoring"
-        ]
         port = <%= node['flyingduck']['port'] %>
-        check = {
```

Comment:
Did you remove the health check on purpose?

Reply:
Yes, that was copied from the onlinefs - the guys are still working on adding a monitoring endpiont to flyingduck. We'll add it back when they are done.